### DISCUSSION TITLE
Can we install this with a typescript template?

### DISCUSSION THREAD

**Discussion by mahmudsg** [react-native-tvos/react-native-tvos#515] [Feature Requests]
Do we have a typescript version of this repo, as React native has this react-native-template-typescript?

**Comment by wouterds** [comment]
Yes, as explained in the README.

**Reply by mahmudsg** [reply]
I already updated my Ruby version to 2.7.5 using rbenv, but when I'm trying to install that typescript template it keeps throwing the following error. 

_Your Ruby version is 2.7.4, but your Gemfile specified 2.7.5_

**Reply by wouterds** [reply]
Seems like a misconfiguration between .ruby-version & Gemfile then, just adjust it so they're both the same and install gems again. We'll look into fixing the template!

**Reply by mahmudsg** [reply]
Thanks! 
I'm a bit new here, where to edit the Gemfile, please?

BTW, I have tried the original react-native-tvos, and it is working well, with the installed ruby version by rbenv.
